src/components/input/math-wrapper.js

```diff
@@ -147,14 +144,9 @@ class MathWrapper {
             }
         } else if (Object.keys(NormalCommands).includes(key)) {
             this._writeNormalFunction(NormalCommands[key]);
-        } else if (key === Keys.FRAC_EXCLUSIVE) {
-            // If there's nothing to the left of the cursor, then we want to
-            // leave the cursor to the left of the fraction after creating it.
```

Comment:
We don't do this anymore because this would prevent us from entering a regular fraction.